tests/unittests/QuantizationTest.cpp

```diff
@@ -879,6 +879,7 @@ TEST_P(Operator, end2endInt8) {
   if (backendSpecificEE.getBackendName() == "OpenCL") {
     keepOriginalPrecisionForNodes.insert(Kinded::Kind::SelectNodeKind);
     keepOriginalPrecisionForNodes.insert(Kinded::Kind::CmpLTENodeKind);
+    keepOriginalPrecisionForNodes.insert(Kinded::Kind::CmpLTNodeKind);
```

Comment:
Why is this necessary? You didn't add OCL support at all, or add CMPLT to testQuantizationEnd2End.

Reply:
I figured in case someone adds it later, but in retrospect they can modify this code also. Will remove.